install/kubernetes/helm/istio/charts/mixer/values.yaml

```diff
@@ -31,6 +31,10 @@ telemetry:
     # based on measurements 100ms p50 translates to p99 of under 1s. This is ok for telemetry which is inherently async.
     latencyThreshold: 100ms
 
+  adapterSecret:
+    enabled: false
+    secretName: adapter-secret
```

Comment:
Same comment as before: do we have any customer that asked for an option to customize the name of the secret ? When we do, we can add it as an option. Otherwise - document the name we picked.

And please don't call it 'adapter-secret' - will be in istio-system with many other secrets. Lots of apps have adapters. 

"mixer-telemetry-secret" ? 

Also I don't believe we need an option for enabled either - as long as it is optional, k8s will just ignore it.

Reply:
For context:  last Fri we had a TOC review of values.yaml - and the number of added options on top of what we had before is scary - so I'm going to push back on any new option unless there is clear evidence a user will want to change the setting.

It is possible to use options with default values ( set the default in the template ), so they don't need to show up in values.yaml.

Reply:
Ok, removed from values. I renamed it to telemetry-adapter-secret.

Reply:
I suppose you mean "adapters" - other adapters may need secrets to connect to whatever they need. Sounds good.